docs/can-guides/commitment/recipes/cta-bus-map/1-setup.js

```diff
@@ -1,17 +1,49 @@
+import { Component } from "//unpkg.com/can@5/core.mjs";
 const proxyUrl = "https://can-cors.herokuapp.com/";
 const token = "?key=piRYHjJ5D2Am39C9MxduHgRZc&format=json";
 const apiRoot = "http://www.ctabustracker.com/bustime/api/v2/"
 const getRoutesEnpoint = apiRoot + "getroutes" + token;
 const getVehiclesEndpoint = apiRoot + "getvehicles" + token;
 
-const BusTrackerVM = can.DefineMap.extend({
-  title: {
-    default: "Chicago CTA Bus Tracker"
-  }
+window.googleAPI = new Promise(function (resolve) {
+  const script = document.createElement("script");
+  script.src = "https://maps.googleapis.com/maps/api/js?key=AIzaSyD7POAQA-i16Vws48h4yRFVGBZzIExOAJI";
+  document.body.appendChild(script);
+  script.onload = resolve;
 });
```

Comment:
Maybe we could move this to the HTML tab instead, since it never gets touched?

Reply:
Done!